server/admin_management/tests/constants.py

```diff
@@ -21,9 +21,10 @@
 TEST_APPLICATION_NAME_FAM = "FAM"
 
 # -------------------- test application admin data ------------------ #
+TEST_APPLICATION_ADMIN_ID = 3
```

Comment:
change the testing application id to 3 (I think it's FOM_TEST), cause most of our default users are already admin of application 1 (FAM) and 2 (FOM_DEV)

Reply:
I think you used it for `application_id`, is it ok to name it `TEST_APPLICATION_ADMIN_APPLICATION_ID `?